python/lsst/consdb/hinfo.py

```diff
@@ -398,6 +398,19 @@ def process_resource(resource: ResourcePath, instrument_dict: dict, update: bool
         conn.commit()
 
 
+def process_local_path(path: str) -> None:
+    # If a directory is provided on the command line, process all yaml
```

Comment:
It's typical to use docstrings instead of single hash comment format for function comments
```suggestion
    """ If a directory is provided on the command line, process all yaml files recursively
    
    Parameters
    -----------
    path : `str`
        Path to directory that contains yaml files
```

Reply:
Yes, DM uses numpydoc for all public functions and methods.  There is tooling available to check the formatting to ensure it is correct. You can see some actions in daf_butler relating to pydocstyle and numpydoc validation and there is a precommit check.